### ISSUE TITLE
More intelligent "ran for"

### ISSUE THREAD

**Issue by brauliobo** [travis-ci/travis-ci#4388]
As it is now, "ran for" time is just a sum of all jobs execution times, as I can verify looking at https://travis-ci.org/coletivoEITA/noosfero-ecosol/builds/71163462

But as travis run jobs in parallel, the real execution perceived by the user is much lower, so I think that the real execution time should be used instead of sum used today.

**Comment by joshk** [comment]
Thank you for creating this issue.

I am actually looking into this and where it fits on our roadmap, but I'm going to close this particular ticket as it is a duplicate of https://github.com/travis-ci/travis-ci/issues/1486

Thanks a bundle
